addons/account/tests/test_transfer_wizard.py

```diff
@@ -0,0 +1,276 @@
+# -*- coding: utf-8 -*-
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+import time
+
+@tagged('post_install', '-at_install')
+class TestTransferWizard(AccountTestInvoicingCommon):
+
+    def setUp(self):
+        super(TestTransferWizard, self).setUp()
+
+        self.company = self.env.ref('base.main_company')
```

Comment:
Such a very old school test ... Adapt the code to be in the setUpClass and use the testing stuff instead (e.g. `company_data` to retrieve all journals / interesting accounts... ).

Reply:
You say 'old', I say 'vintage'! 

I'll have a look :p